pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -162,7 +167,7 @@ def work(self, response_dict=None):
         # log encounter
         self.emit_event(
             'pokemon_appeared',
-            formatted='*A wild {} appeared!* (CP: {}) (NCP: {}) (Potential {}) (A/D/S {})'.format(pokemon.name, pokemon.cp, round(pokemon.cp_percent, 2), pokemon.iv, pokemon.iv_display),
+            formatted='A wild {} appeared! (CP: {}) (IV: {}) (A/D/S {}) (NCP: {})'.format(pokemon.name, pokemon.cp,  pokemon.iv, pokemon.iv_display, round(pokemon.cp_percent, 2),),
```

Comment:
Doesn't need all those parentheses. Change to

(CP: {} IV: {} A/D/S {} NCP: {})